lizard_auth_client/tests.py

```diff
@@ -248,6 +249,131 @@ def test_create_from_list_of_dicts(self):
+                    'code': 'superhero',
+                    'name': 'Spidermanplus',
+                    'external_description': 'Spider+',
+                    'internal_description': 'Fake+',
+                }
+            },
+                {
+                'organisation': {
+                    'unique_id': u"77f5a464c35044c19bc7d4b42d7f58da",
+                    'name': "Power Inc."
+                },
+                'role': {
+                    'unique_id': 'superpower',
+                    'code': 'Hero',
+                    'name': 'Spiderman',
+                    'external_description': 'Spider',
+                    'internal_description': 'Fake',
+                }
+            },
+            ])
+
+    def test_user_org_role_dict_has_user_info(self):
+        """
+        make sure the user_org_role dict contains the user data
+        """
+        payload_dict = get_user_org_role_dict(self.user)
+        self.assertEqual(payload_dict['username'], 'testuser')
+        self.assertFalse(payload_dict['is_superuser'])
+        self.assertEqual(payload_dict['email'], 'testuser@beingused.to')
+
+    def test_user_org_role_dict_contains_roles(self):
+        """
+        a user can have different roles for different organisations
+        """
+        expected_roles_velen_huurmans = [u'Hero', u'superhero']
+        expected_roles_power_inc = [u'Hero']
+        payload_dict = get_user_org_role_dict(self.user)
+        self.assertEqual(len(payload_dict['organisations']), 2)
+        # should have two permissions
+        self.assertEqual(
+            len(payload_dict['organisations'][0]['permissions']), 2
+        )
+        self.assertIn(
+            payload_dict['organisations'][0]['permissions'][0],
+            expected_roles_velen_huurmans
+        )
+        self.assertIn(
+            payload_dict['organisations'][0]['permissions'][1],
+            expected_roles_velen_huurmans
+        )
+        # should only contain permission "Hero"
+        self.assertEqual(
+            len(payload_dict['organisations'][1]['permissions']), 1
+        )
+        self.assertIn(
+            payload_dict['organisations'][1]['permissions'][0],
+            expected_roles_power_inc
+        )
+
+    def test_user_org_role_is_connected_is_excluded(self):
+        """
+        the roles a user can have for an organisation might
+        also include the role "is_connected". For the payload
+        this role is irrelevant so it has to be filtered out
+        """
+        role = models.Role.create_from_dict({
+            'unique_id': 'connected',
+            'code': 'is_connected',
+            'name': 'connector',
+            'external_description': 'connected',
+            'internal_description': 'connected',
+        })
+        # check if it is in the DB
+        ic = models.Role.objects.get(code='is_connected')
+        self.assertTrue(repr(ic))
+
+        orga = models.Organisation.objects.get(
```

Comment:
Ook hier niet afkorten. In de hele codebase heet het `organisation`. Anders begint de een met `org`, de volgende `orga`, enz. Je moet niet na hoeven denken: een model `Organisation` heeft objecten die `organisation` heten. In principe.